code/__DEFINES/~skyrat_defines/layers.dm

```diff
@@ -0,0 +1,5 @@
+///Plane of the "splash" icon used that shows on the lobby screen. Nothing should ever be above this. (Except the logo)
+#define BLACK_FADE 9997
+#define SKYRAT_LOGO 9999
+
+#define CHAT_INDICATOR_LAYER 6 // 1 above BYOND's FLY_LAYER
```

Comment:
Do we still need this since tg added typing indicators?

Reply:
no

Reply:
can we get rid of it then